samples/modelbuilder/ObjectDetection_StopSigns/StopSignDetectionML.Model/ConsumeModel.cs

```diff
@@ -53,7 +47,7 @@ private static void DownloadModel()
         {
```

Comment:
Can you delete the code that downloads the model here?

Reply:
Whoops yes! Fixed with https://github.com/dotnet/machinelearning-samples/pull/882/commits/0a55fa5ebcb0c763c125bc17c511d57f56f59f9a